src/helm/benchmark/annotation/bigcodebench_annotator.py

```diff
@@ -0,0 +1,109 @@
+from typing import Any, List
+
+from helm.benchmark.adaptation.request_state import RequestState
+from helm.benchmark.annotation.annotator import Annotator
+from helm.common.request import Request
+from gradio_client import Client, handle_file
+from tempfile import TemporaryDirectory
+
+from helm.common.hierarchical_logger import hlog
+
+import ast
+import traceback
+import time
+import json
+
+
+def syntax_check(code, verbose=False):
+    try:
+        ast.parse(code)
+        return True
+    except (SyntaxError, MemoryError):
+        if verbose:
+            traceback.print_exc()
+        return False
+
+
+def code_extract(text: str) -> str:
```

Comment:
I presume this is what the original paper did? It's really slow...

Reply:
Yes it's from the original paper and it's indeed pretty slow..